modules/tribes/client/components/TribesList.component.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styled from 'styled-components';
+
+import Tribe from './Tribe';
+import SuggestTribe from './SuggestTribe';
+
+const List = styled.ul`
+  margin: -5px -5px 15px -5px;
+  line-height: 0;
+`;
+
+const Item = styled.li`
+  line-height: 1.42857143; // set back the line-height set by bootstrap
+  padding: 5px;
+  display: inline-block;
+  vertical-align: bottom;
+  // background-color: red;
+  @media (max-width: 615.9999px) {
+    width: 100%;
+  }
+
+  @media (min-width: 616px) and (max-width: 991.9999px) {
+    width: 50%;
+  }
+
+  @media (min-width: 992px) {
+    width: 33.3333%;
+  }
+`;
+
+export default function TribesList({ tribes, user, onMembershipUpdated }) {
+  return (
+    <List className="list-unstyled tribes-grid">
```

Comment:
I wonder how to handle existing CSS classes as we move forward? Should it all move into styled-component definitions?

Reply:
I don't know. What do you think?
I'm fine with leaving until we have the definitive decision whether to go forward with separate stylesheets or styled-components or other way of managing styles. Less work for now... :slightly_smiling_face:

Reply:
Yup, I agree to leave it for now.